guides/tutorials/nextjs-openai-chatbot.mdx

```diff
@@ -0,0 +1,592 @@
+---
+title: Chatbot example with OpenAI, Next.js and Literal AI
+---
+
+The goal of this tutorial is to set up a simple chatbot and give it superpowers using Literal AI.
+
+We will explore the following functionalities of the Literal AI platform :
```

Comment:
```suggestion
We will explore the following functionalities of the Literal AI platform:
```

Reply:
oh god you busted me 🫣 yes that is a bad habit i have a hard time correcting